convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -426,53 +426,39 @@ def test_call_yum_cmd_with_repo_overrides(self):
             ],
         )
 
-    def test_get_pkgs_to_distro_sync(self):
-        problematic_pkgs = {
-            "protected": set(["a"]),
-            "errors": set(["b"]),
-            "multilib": set(["c", "a"]),
-            "required": set(["d", "a"]),
-            "mismatches": set(["e", "a"]),
-        }
-        all_pkgs = pkghandler.get_pkgs_to_distro_sync(problematic_pkgs)
-        self.assertEqual(
-            all_pkgs,
-            problematic_pkgs["errors"]
-            | problematic_pkgs["protected"]
-            | problematic_pkgs["multilib"]
-            | problematic_pkgs["required"],
-        )
-
-    @unit_tests.mock(pkghandler, "call_yum_cmd", CallYumCmdMocked())
-    def test_resolve_dep_errors_one_downgrade_fixes_the_error(self):
-        pkghandler.call_yum_cmd.fail_once = True
-
-        pkghandler.resolve_dep_errors(YUM_PROTECTED_ERROR, set())
-
-        self.assertEqual(pkghandler.call_yum_cmd.called, 1)
-
-    @unit_tests.mock(pkghandler, "call_yum_cmd", CallYumCmdMocked())
-    def test_resolve_dep_errors_unable_to_fix_by_downgrades(self):
-        pkghandler.call_yum_cmd.return_code = 1
-        pkghandler.call_yum_cmd.return_string = YUM_MULTILIB_ERROR
-
-        pkghandler.resolve_dep_errors(YUM_PROTECTED_ERROR, set())
-
-        # Firts call of the resolve_dep_errors, pkgs from protected error
-        # are detected, the second call pkgs from multilib error are detected,
-        # the third call yum_cmd is not called anymore, because the
-        # problematic packages then remain the same (simulating that the
-        # downgrades do not solve the yum errors)
-        self.assertEqual(pkghandler.call_yum_cmd.called, 2)
+    class GetInstalledPkgsWFingerprintsMocked(unit_tests.MockFunction):
```

Comment:
This seems to me as an accidental revert of changes from https://github.com/oamg/convert2rhel/commit/5666d74604582665281194819cbfb54131747073#diff-c3ce6b0e1e7ddae0678f9eb7c71f9c5f8aecbfd5b7db1deb7d8947ccf1492a08L369.

Reply:
yeah, it was an accidental revert caused by the rebase. It is not there anymore.